### PR TITLE
Created file to read games csv file 

### PR DESCRIPTION
**Pull request by fieldstyler**
PR by Tyler F

Create game data array.

### CODE REVIEW

**Comment by priyapower** [comment]
Reviewer: Priya Power

File: lib/game_data.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class GameData
+
+  attr_reader :game_id, :season, :type,
+      :date_time, :away_team_id, :home_team_id,
+      :away_goals, :home_goals, :venue, :venue_link
+
+    def initialize()
```
Comment (priyapower) [inline lib/game_data.rb:7]: These instance variables are used to create array for Module

File: lib/game_data.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      @home_team_id = home_team_id
+      @away_goals = away_goals
+      @home_goals = home_goals
+      @venue = venue
+      @venue_link = venue_link
+    end
+
+    def create_attributes(table, line_index)
+      index = 0
+      @game_id = table[line_index][index]
```
Comment (priyapower) [inline lib/game_data.rb:22]: Used for creating array of game data for access later

File: test/game_data_test.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      games: game_path,
+      teams: team_path,
+      game_teams: game_teams_path
+    }
+
+    @game_data = GameData.new
+    @table = CSV.parse(File.read('./data/dummy_file_games.csv'), headers: true)
+  end
+
+  def test_it_exists
```
Comment (priyapower) [inline test/game_data_test.rb:23]: I see you tested it exists

File: test/game_data_test.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    assert_instance_of GameData, @game_data
+  end
+
+  def test_it_can_create_many_objects
```
Comment (priyapower) [inline test/game_data_test.rb:27]: I see you tested your create_attributes method

**Comment by fieldstyler** [comment]
Resolved by Tyler

**Pull request merged**